src/components/ZNALink/ZNALink.tsx

```diff
@@ -16,7 +16,7 @@ const ZNALink: React.FC<ZNAProps> = ({ className, domain, style }) => {
 			className={`${styles.ZNALink} ${className ? className : ''}`}
 			style={style}
 		>
-			<span style={{ cursor: 'default', opacity: 0.75 }}>0://</span>
+			<a href='https://www.zero.tech/'  target="_blank" rel="noreferrer" style={{ cursor: 'cursor', opacity: 0.75, color: 'white' }}>0://</a>
```

Comment:
There's a class for links `.alt-link` which makes them white - switch it over to use this rather than setting color in style. Again, this is a very minor nitpick

Reply:
Done